### DISCUSSION TITLE
Version Mismatch Help

### DISCUSSION THREAD

**Discussion by CamManCraft2001** [EssentialsX/Essentials#4160] [Q&A] [answered]
When I do /ess version I get a message saying that my EssentialsSpawn is mismatched as well as my server version. Here is what it says:

Server version: 1.16.5-R0.1-SNAPSHOT git-Paper-692 (MC: 1.16.5)
EssentialsX version: 2.18.2.0
PlaceholderAPI version: 2.10.9
LuckPerms version: 5.3.21
Vault version: 1.7.3-b131
EssentialsXSpawn version: 2.19.0-dev+102-edf664e
Citizens version: 2.0.27-SNAPSHOT (build 2217)
Version mismatch! Please update all Essentials jars to the same version.
You are running an unsupported server version!

Can someone link me to where I can download the version for EssentialsSpawn that is supported? Thanks!

**Comment by JRoy** [comment, marked as answer]
You have a different Essentials version than EssentialsSpawn version, download the latest Essentials and EssentialsSpawn builds from [https://essentialsx.net/downloads.html](https://essentialsx.net/downloads.html)

**Comment by CamManCraft2001** [comment]
Ok, I fixed it Thanks!
